### DISCUSSION TITLE
Two-factor authentication activation process issue

### DISCUSSION THREAD

**Discussion by yvan-solutions** [community/community#66864] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

While enabling 2FA on one of my accounts, at the second step titled "Download your recovery codes", I printed the codes and do not want to download them at all. 

I cannot go to step 3 without clicking the "Download" button, which goes against my personal security policy.

Also the button to continue is labelled "I have downloaded the codes" (I think I remember that was it) which might confuse people and explain question/issues [like this one](https://github.com/orgs/community/discussions/66421).

I figured it out but it might not be that obvious for all folks.

**Comment by khuramgill** [comment]
It's great to see that you're proactive about your security and recovery codes when enabling Two-Factor Authentication (2FA) on your account. However, GitHub's 2FA setup process might indeed be a bit confusing for some users, as the wording and actions may not align with personal preferences or security practices.

Here are some insights and suggestions:

1. **Printed Recovery Codes**: If you've printed your recovery codes and do not wish to download them digitally, that's a valid security choice. However, the GitHub interface might not make this entirely clear during the setup process.

2. **Button Wording**: The button wording, as you mentioned, could be improved to make it more explicit. Instead of saying "I have downloaded the codes," it could be something like "I have securely stored the recovery codes" to better reflect that you've printed them.

3. **Provide Feedback to GitHub**: GitHub is typically receptive to user feedback, especially when it concerns usability and security. You can consider providing feedback to GitHub about your experience with the 2FA setup process. They might take your input into account for future improvements.

4. **Documentation**: GitHub could enhance its documentation to explain the flexibility in handling recovery codes. Users should be made aware that they have the option to print codes and keep them physically rather than downloading them digitally.

5. **User Education**: Educating users about the importance of recovery codes and different storage methods can also be beneficial. GitHub could include information or links to articles about best practices for 2FA and recovery code management.

In the meantime, while waiting for potential improvements, it's crucial to continue following your own security practices. Storing printed recovery codes in a safe and accessible location is a reasonable approach, as long as you ensure they are kept secure and not accessible to unauthorized individuals.

Remember that GitHub's security features are designed to provide flexibility to accommodate a wide range of user preferences and practices, so you're not alone in customizing the setup to your specific needs.

**Comment by DNin01** [comment]
I understand your concern. During the initial setup, you do have to download the codes to continue. However, later on, you can [regenerate them](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication-recovery-methods#generating-a-new-set-of-recovery-codes) and then you should be able to print them instead.

**Comment by Kherae** [comment]
Similar situation. Using firefox if that means anything. I cannot get past step 2, because the 'I have saved my recovery codes' button only is usable for a brief moment after clicking 'download' which loads the 'download' in that page. naturally, hitting the 'back' button puts me back at square one. 
I immensely dislike the mandatory push for 2FA, and this is just making it that much worse.

**Comment by dinhtrungndt** [comment]
Report #1
